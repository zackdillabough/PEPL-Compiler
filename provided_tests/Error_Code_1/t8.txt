POLY F1(X) = X + 1;
POLY F2 = x^2 + x + 1;
POLY F1(A, B) = A - A + B;
POLY F3 = x;
POLY F4 = x + 1;
POLY F2(X) = X^5050 - 38792647;
START
INPUT X;
INPUT Y;
F3(Y);
F4(F3(Y));
1 2 7652 297673
