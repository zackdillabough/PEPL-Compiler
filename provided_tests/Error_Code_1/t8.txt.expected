Error Code 1: 1 2 3 6 